stratum/hal/lib/tdi/es2k/es2k_sde_table_data.cc

```diff
@@ -68,47 +68,44 @@ ::util::Status TableData::GetPktModMeterConfig(
     TdiPktModMeterConfig& cfg) const {
   // Condition checks for Indirect PacketModMeter
   if (cfg.isPktModMeter) {
-    RETURN_IF_ERROR(GetField(*(table_data_.get()), kMeterCirPps, &cfg.cir));
-    RETURN_IF_ERROR(GetField(*(table_data_.get()), kMeterCommitedBurstPackets,
-                             &cfg.cburst));
-    RETURN_IF_ERROR(GetField(*(table_data_.get()), kMeterPirPps, &cfg.pir));
+    RETURN_IF_ERROR(GetField(*table_data_.get(), kMeterCirPps, &cfg.cir));
```

Comment:
Nuts. I only just noticed that `GetField` is being called in each instance with `*table_data_.get()`.

You could add `auto tbl_data = *table_data_.get();` right after the `if` and then change the parameter instances to `*tbl_data`.

**READABILITY** (and possibly an optimization)